README.md

```diff
@@ -0,0 +1,25 @@
+# terra-axon-examples
+
+## Overview
+
+This repository contains example notebooks and documentation for working with [Enterprise Terra](https://verily.com/solutions/terra/).
+
+[Enterprise Terra](https://verily.com/solutions/terra/) is a new platform developed by Verily to connect the multimodal data ecosystem and meet the needs of commercial or industry-led research organizations. Building on Terra’s Academic roots, Enterprise Terra is an integrated platform that provides the secure foundation to discover, govern, and unify multimodal data into a collaborative environment to accelerate discoveries.
+
+- Explore: discover multimodal data sets available in Enterprise Terra’s data ecosystem.
+- Unify: connect all data types, from both internal and external sources, into a single platform and under one governance framework.
+- Analyze: unlock insights faster with familiar and proprietary analysis tools.
+- Collaborate: share best practices and publish reproducible research.
+- Scale: maximize the power of any cloud with security, compliance, and support from industry standards.
```

Comment:
'any cloud' - are you referring to Google, AWS, and Azure?  Might be good to specify.

Reply:
IMO we need to be specific here. If a customer asks about a cloud we don't currently support it'd be good to have that feedback.